swagger.json

```diff
@@ -1324,7 +1324,7 @@
                     "format": "int32"
                 },
                 "plug_connected_boiler": {
-                    "type": "boolean"
+                    "type": "int32"
```

Comment:
I think that the `NAPlug.plug_connected_boiler` field needs the change from boolean to int, too.
I've got conversion issues with gson until I changed it too...